docs/src/modules/ROOT/pages/quickstart/shared/school-timetabling/school-timetabling-model.adoc

```diff
@@ -269,6 +269,12 @@ public class Lesson {
         this.studentGroup = studentGroup;
     }
 
+    public Lesson(String id, String subject, String teacher, String studentGroup, Timeslot timeslot, Room room) {
```

Comment:
I would add a comment this constructor is only for tests and make the constructor "package private" (no access modifier).